dusty/compiler/nginx/__init__.py

```diff
@@ -1,10 +1,12 @@
 from ...constants import NGINX_MAX_FILE_SIZE, VM_IP_FROM_DOCKER
 
-def _nginx_proxy_string(port_spec):
-    return "proxy_pass http://{}:{};".format(VM_IP_FROM_DOCKER, port_spec['proxied_port'])
+def _nginx_proxy_string(port_spec, include_http=True):
```

Comment:
Yeah it was the default so I didn't need to include it. Anyway, took your suggestion